cpp/conv_templates.cc

```diff
@@ -295,6 +295,25 @@ Conversation CodeGPT() {
   return conv;
 }
 
+Conversation WizardLM() {
+  Conversation conv;
+  conv.name = "wizardlm";
+  conv.system = "{instruction}\n\n### Response:" ;
```

Comment:
Is `{instruction}` a placeholder?

Reply:
I suppose no. See the README of https://github.com/nlpxucan/WizardLM.

Reply:
After double checking I think it is.. https://github.com/nlpxucan/WizardLM/blob/main/WizardLM/src/inference_wizardlm.py#L57-L61